src/components/inputs/radioButtons/rectangularRadioButton.js

```diff
@@ -24,6 +24,34 @@ function borderColor(props) {
   return props.theme.colors.gray4
 }
 
+const handleShapeStyles = shape => {
```

Comment:
Why does the `RectangularRadioButton` need shapes too? Wouldn't we just use the `RadioButton` component with `shape='rectangular'`?

Reply:
Duh I just realized one's a radio button and another one a checkbox, sorry about the confusion! 😅